plugins/vcpu_count/spec.rb

```diff
@@ -0,0 +1,112 @@
+require_relative './plugin.rb'
+describe 'VCPUCount plugin' do
+  before(:all) do
+    @db_table = Iam.DB[:vcpu_count_measurements]
+  end
+
+  # Register method
+  describe '.register method' do
+    it 'does not raise an error when invoked' do
+      expect { VCPUCount.new.register }.to_not raise_error
+    end
+
+    it 'creates a vcpu_count_measurements table' do
+      # Table shouldn't exist before registration
+      expect do
+        Iam.settings.DB.table_exists?(:vcpu_count_measurements).to be_false
+      end
+
+      VCPUCount.new.register
```

Comment:
Is there a reason we have to explicitly call the `register` method on the plugin? I recall we talked about calling the register method by default when a Plug-in object is created.